meetings/2018-10-26.md

```diff
@@ -0,0 +1,68 @@
+* Dan: Chris do you still want to leverage the user feedback space and opportunity to support tooling group, or tolling achieved what it needs and that’s going to be a new home?
+* Chris: I’m not sure, I talked about this with the group at the club summit and we decided to table that for now, we have plenty of things to do, stuff that needs to get done. I would expect to do another one at some point, I don’t know if it’s going to be a regular occurrence, I guess not, but I can’t say quite yet if we want to move that effort back into that group.
+* Ahmad: Enterprise still need a few more tweaks before we can talk about how did that evolved. I don’t see the kind of enterprise group being as active as part of the same community, and somehow it might end up needing a sub community of its own, especially when it comes like call to actions and getting them involved in initiatives that the core team would like to run through for example testing a beta release or getting feedback o performance, either we can replicate and we can make a little corner of it, that we can add these people to without becoming a point of spam and  noise  to them, as those folks who will be part of the enterprises and new companies in the future might be interested in helping out, they don’t want to be overwhelmed either, but I think could be as simple as a newsletter in a call tp action every once in a while when needed and we want to do like these kind of meetings , it’s a personal invite that works best to get these people to give us their time.
+* Dan: The Enterprise group would only hear from their focus area?
+* Ahmad: My next action is to schedule another community meeting for the enterprise group, but as a requirement for that I think like having that email signup / newsletter/ method of communication, beyond just me sending emails to individuals so people can opt into that conversation and then we can make it easier for follow-ups, especially as we want to start asking for the different working groups what actions they might want to surface whether it’s a survey or  an engagement of some sort so we can make it more structured. Again, the newsletters how would that work? Who would be the person to help?
+* Tierney: Zibby. If we want to use the existing work like the prior art that would be the LF mailing list.
+* Michael: You want an infrastructure that will manage people locking in and out of their mailing list, I guess.
```

Comment:
Michael: You want an infrastructure that will manage people locking in and out of their mailing list, I guess.

>>>
Michael: You want an infrastructure that will manage people getting in and out of the mailing list, I guess.